src/patterns/SaleHeaderBanner/SaleHeaderBanner.tsx

```diff
@@ -0,0 +1,106 @@
+import { ComponentProps, forwardRef } from 'react';
+import { getCommonProps } from '../../utils';
+import classnames from 'classnames';
+import { Text, TextVariants } from '../../components/Text';
+import Button from '../../components/Button/Button';
+import { PageContentWrapper as PageMargin } from '../../components/PageContentWrapper';
+import { Link } from '../../components/Link';
+import { AuctionState } from './types';
+
+// You'll need to change the ComponentProps<"htmlelementname"> to match the top-level element of your component
+export interface SaleHeaderBannerProps extends ComponentProps<'header'> {
+  /**
+   * What is the title of the auction?
+   */
+  auctionTitle: string;
+  /**
+   * Where is the auction taking place?
+   */
+  location: string;
+  /**
+   * Depending on auction state, When does the auction open or close
+   */
+  date: string;
+  /**
+   * Clarifies the date based on the auction state
+   */
+  occuranceLabel: string;
+  /**
+   * What is the current state of the auction?
+   */
+  auctionState: AuctionState;
+  /**
+   * The URL of the banner image
+   */
+  imageSrcUrl: string;
+}
+/**
+ * ## Overview
+ *
+ * Sale header banner component, supports 3 states of the auction: pre-sale, open for bidding, and closed.
+ *
+ * [Figma Link](https://www.figma.com/design/OvBXAq48blO1r4qYbeBPjW/RW---Sale-Page-(PLP)?node-id=1-6&m=dev)
+ *
+ * [Storybook Link](https://phillips-seldon.netlify.app/?path=/docs/patterns-saleheaderbanner--overview)
+ */
+
+const SaleHeaderBanner = forwardRef<HTMLElement, SaleHeaderBannerProps>(
+  ({ auctionTitle, date, location, auctionState, imageSrcUrl, occuranceLabel, className, ...props }, ref) => {
+    const { className: baseClassName, ...commonProps } = getCommonProps(props, 'SaleHeaderBanner');
+
+    const isOpenForBidding = auctionState === AuctionState.openForBidding;
+    const isClosed = auctionState === AuctionState.past;
+
+    return (
+      <header {...commonProps} className={classnames(baseClassName, className)} {...props} ref={ref}>
+        <img
+          id="PLACEHOLDER FOR IMAGE COMPONENT"
+          alt={auctionTitle}
+          className={`${baseClassName}__image`}
+          src={imageSrcUrl}
+        ></img>
+        <PageMargin>
+          <div className={`${baseClassName}__stack`}>
+            {isOpenForBidding && (
+              <div id="PLACEHOLDER FOR TIMER COMPONENT" className={`${baseClassName}__countdown-container`}>
+                <Text variant={TextVariants.heading5}>Lots Close in</Text>
+                <Text variant={TextVariants.heading5}>2 Days</Text>
+                <Text variant={TextVariants.heading5}>17 Hours</Text>
+              </div>
+            )}
```

Comment:
I wonder if we should make this composable? These string will need a way to be translated. We either need to include as props for this component or allow for composability of a timer that will include translated strings.

Reply:
wondering if this actually makes sense as a seldon component, rather than living in remix as a composition of different seldon components (which are already composable)

or will this be used by phillips public?

Reply:
I think there are some benefits to it being in Seldon, first of all, the Easel Design System from Figma references it in their Pattern section.  I think we need to examine some of our other components in Remix (LotNavigation, etc) for potentially moving to Seldon as well.